docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -284,10 +284,10 @@ Where *closedBoxId* is the id of the closed box and *boxKey* is correct proof fo
 Custom Proof / Proposition creation
 ###################################
 
-A proposition is a locker for a box, and Proof is an unlocker for a box. For some reason, a way how the box is locked/unlocked can be changed by the SDK developer. For example, a special box can be opened by two or more independent private keys. For such reason, custom Proof / Proposition can be created.
+A proposition is a locker for a box, and Proof is an unlocker for a box. For some reason, the manner in which the box is locked/unlocked can be changed by the SDK developer (should this be sidechain developer?). For example, a special box can be opened by two or more independent private keys. For such reason, custom Proof/Proposition can be created.
 
 * Creating custom Proposition
-  For creating a custom Proposition  ``ProofOfKnowledgeProposition<S extends Secret>`` interface shall be implemented. Generic     parameter is just a marker for the type of private key, for example, *PrivateKey25519* It could be used. Inside the Proposition, we could put two different public keys, which are used for locking the box.
+  For creating a custom Proposition  ``ProofOfKnowledgeProposition<S extends Secret>`` interface shall be implemented. Generic parameter is just a marker for the type of private key. For example, *PrivateKey25519* could be used. Inside the Proposition, we could use two different public keys to lock the box.
```

Comment:
To create a custom proposition  ``ProofOfKnowledgeProposition<S extends Secret>`` interface shall be implemented. The generic parameter is a marker for the type of private key. For example, *PrivateKey25519* could be used. Inside the proposition, we could use two different public keys to lock the box.